spec/lib/rabbit_feed/configuration_spec.rb

```diff
@@ -11,6 +11,18 @@ module RabbitFeed
       subject { (described_class.new options).queue }
 
       it { should eq 'test.rabbit_feed' }
+
+      context 'when RabbitFeed.route_prefix_extension is set' do
```

Comment:
`RabbitFeed.route_prefix_extension` no longer exists - could change to `when a route_prefix_extension is set`